src/applications/mhv/shared/hooks/useDatadogRum.jsx

```diff
@@ -20,12 +20,17 @@ const initializeDatadogRum = config => {
   }
 };
 
-const useDatadogRum = config => {
+const useDatadogRum = (config, user) => {
   useEffect(
     () => {
       initializeDatadogRum(config);
+      if (user?.login?.currentlyLoggedIn) {
+        datadogRum.setUser({
+          id: user.profile.accountUUid,
```

Comment:
do we expect any possibility that `user.profile.accountUUid` would be missing or not set here?

Reply:
I don't think so, but I added in a default value to see if it ever shows up.